config/manager/manager.yaml

```diff
@@ -52,9 +47,10 @@ spec:
           periodSeconds: 10
         resources:
           limits:
-            cpu: 50m
-            memory: 700Mi
+            cpu: 200m
+            memory: 100Mi
```

Comment:
What is the point of updating those resources config?

It is true that they can be updated via Operator subscription, but given the fact we are the only users of the operator, we are sharing the subscription at hive, I wouldn't add high restrictions and configure correct generic resources here:
- I have checked on dev-eng, memory is stable at 140Mb with peaks at 650MB on startup
- I have checked on stg-saas, memory is stable at 230Mb with peaks at 352MB (and maybe more on startup),
- I have checked on pro-saas, memory is stable at 140Mb with peaks at 160MB (and maybe more on startup)

So, to avoid OOMkiller on pod startup (need to create a worker for every controller...), I would minimum configure something like:
- Req: 150 Mi
- Limit: 700Mi 

What do you think? cc  @raelga